build/docker-exec

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]] ; then
+    echo "build/docker-exec - run a rake command on a specified ruby version"
+    echo "Usage:"
+    echo "  build/docker-exec 2.2.5 spec"
+    exit 1
+fi
+
+ruby_version="$1"
+rake_task="$2"
+
+cp -r /app /src
+cd /src/app || exit 1
+
+# Disable strict settings that break chruby
+set +o nounset
+set +o noglob
+
+source "/usr/local/share/chruby/chruby.sh"
```

Comment:
I would tend to use single quotes on any strings where I am not doing variable interpolation.